sdk/formrecognizer/Azure.AI.FormRecognizer/src/CopyModelOperation.cs

```diff
@@ -153,14 +166,17 @@ private async ValueTask<Response> UpdateStatusAsync(bool async, CancellationToke
 
                 if (update.Value.Status == OperationStatus.Succeeded)
                 {
-                    _hasCompleted = true;
+                    // we need to first assign a vaue and then mark the operation as completed to avoid race conditions
```

Comment:
nit: vaue => value

Reply:
Maybe being more specific about what race condition we're talking about? I'm afraid this information will be lost if we don't document it properly:
```
We need to first assign a value and then mark the operation as completed to avoid a race condition with the getter in Value
```
These comments applies to all `***Operation` types.